jetty-server/src/test/java/org/eclipse/jetty/server/handler/IPAccessHandlerTest.java

```diff
@@ -57,30 +57,44 @@
     private static NetworkConnector _connector;
     private static IPAccessHandler _handler;
 
-    @BeforeAll
-    public static void setUp()
+    @BeforeEach
```

Comment:
Now that you have changed the test annotation to be per-instance, all fields should not be `static`: `_server`, `_connector`, `_handler`.